frontend/src/concepts/pipelines/content/compareRuns/CompareRunTableToolbar.tsx

```diff
@@ -0,0 +1,96 @@
+import PipelineVersionSelect from '~/concepts/pipelines/content/pipelineSelector/CustomPipelineVersionSelect';
+import { PipelineRunVersionsContext } from '~/pages/pipelines/global/runs/PipelineRunVersionsContext';
+
+export type FilterProps = Pick<
+  React.ComponentProps<typeof PipelineFilterBar>,
+  'filterData' | 'onFilterUpdate' | 'onClearFilters'
+>;
+
+const CompareRunTableToolbar: React.FC<FilterProps> = ({ ...toolbarProps }) => {
+  const { versions } = React.useContext(PipelineRunVersionsContext);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { [RuntimeStateKF.RUNTIME_STATE_UNSPECIFIED]: unspecifiedState, ...statusRuntimeStates } =
+    runtimeStateLabels;
+
+  const options = React.useMemo(
+    () => ({
+      [FilterOptions.NAME]: 'Run',
+      [FilterOptions.EXPERIMENT]: 'Experiment',
+      [FilterOptions.PIPELINE_VERSION]: 'Pipeline version',
+      [FilterOptions.CREATED_AT]: 'Started',
+      [FilterOptions.STATUS]: 'Status',
+    }),
+    [],
+  );
+
+  return (
+    <PipelineFilterBar
+      {...toolbarProps}
+      filterOptions={options}
+      filterOptionRenders={{
+        [FilterOptions.NAME]: ({ onChange, ...props }) => (
+          <TextInput
+            {...props}
+            data-testid="search-for-run-name"
+            aria-label="Search for a run name"
+            placeholder="Search..."
+            onChange={(_event, value) => onChange(value)}
+          />
+        ),
+        [FilterOptions.EXPERIMENT]: ({ onChange, value, label }) => (
+          <ExperimentSearchInput
+            onChange={(data) => onChange(data?.value, data?.label)}
+            selected={value && label ? { value, label } : undefined}
+          />
+        ),
+        [FilterOptions.PIPELINE_VERSION]: ({ onChange, label }) => (
+          <PipelineVersionSelect
+            versions={versions}
+            selection={label}
+            onSelect={(version) => onChange(version.pipeline_version_id, version.display_name)}
+          />
+        ),
+        [FilterOptions.CREATED_AT]: ({ onChange, ...props }) => (
+          <DashboardDatePicker
+            {...props}
+            hideError
+            aria-label="Select a start date"
+            onChange={(_, value, date) => {
+              if (date || !value) {
+                onChange(value);
+              }
+            }}
+          />
+        ),
+        [FilterOptions.STATUS]: ({ value, onChange, ...props }) => (
+          <SimpleDropdownSelect
+            {...props}
+            value={value ?? ''}
+            aria-label="Select a status"
+            options={Object.values(statusRuntimeStates).map((v) => ({
+              key: v,
+              label: v,
+            }))}
+            onChange={(v) => onChange(v)}
+            data-testid="runtime-status-dropdown"
+          />
+        ),
+      }}
+    >
+      <ToolbarItem>
+        {/* TODO: Add manage runs modal */}
+        <Button>Manage runs</Button>
```

Comment:
can you just make this link to `comapareRuns/add?runs=...` Jeff's PR will go in first and he created the add page (not modal) so it just links. There should be an example in the empty/error states for compare runs